Classes/Issues/GithubClient+Issues.swift

```diff
@@ -289,13 +289,21 @@ extension GithubClient {
             return
         }
 
-        // https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
+        // https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level
         request(Request(
-            path: "repos/\(owner)/\(repo)/collaborators/\(viewer)",
+            path: "repos/\(owner)/\(repo)/collaborators/\(viewer)/permission",
             headers: ["Accept": "application/vnd.github.hellcat-preview+json"],
```

Comment:
Does this endpoint still need the accepts value?

Reply:
Good catch, nope!